datasets/emit-landfill.data.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: landfill-emit
+name: "EMIT Landfill"
+description: "FILL IN"

```

Comment:
Does this mean to be this value?

Reply:
It shows up in the Catalog view.
![Screen Shot 2024-05-13 at 4 23 26 PM](https://github.com/NASA-IMPACT/veda-config/assets/4583806/8cbc096d-a322-4fca-abe3-8c81bc8e7e30)